lib/verifiablePresentation/SSI.types.ts

```diff
@@ -41,7 +41,7 @@ export interface Presentation {
   type: string[];
   verifiableCredential: VerifiableCredential[];
   presentation_submission?: PresentationSubmission;
-  holder: string;
+  holder: string[];
```

Comment:
presentation must have a single `holder` value

Reply:
Fixed!

Reply:
Holder is optional.  Please always check against specs!

Reply:
Fixed!